guides/concepts/networks.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Networks
+---
+
+Stellar has two public networks: the Public Network (pubnet), which is the main decentralized
+network used throughout the world, and the Test Network (testnet), a network maintained by the
+Stellar Development Foundation that is available for developers to test their applications. For
+more in-depth information on testnet, [see our our additional documentation](test-net.md).
+
+## Network Passphrases
+
+Each Stellar network has its own unique passphrase, which is used when validating signatures on a
+given transaction. If you sign a transaction for one network but submit it to another, it won't be
+considered valid. The format of a passphrase is typically `'[Network Name] ; [Month of Creation]
+[Year of Creation]'`.
+
+The current passphrases for the Stellar pubnet and testnet are:
+
+* Pubnet: `'Public Global Stellar Network ; September 2015'`
+* Testnet: `'Test SDF Network ; September 2015'`
+
+The passphrase is utilized for two reasons:
+
+* The network passphrase is used as the seed for the root account (master network key) at genesis.
+* For building hashes of transactions, which are ultimately what is signed by each signer's secret
+  key in a transaction envelope. Again, this allows you to verify that a transaction was intended
+  for a specific network by its signers.
+
+Most SDKs have the passphrases hardcoded for the Stellar pubnet and testnet, but if you're running
+a private network, you'll need to manually pass in a passphrase to be used whenever transaction
+hashes are generated. All of Stellar's official SDKs give you the ability to use a network with a
+custom passphrase.
+
+### Moving To Production
+
+When creating your application on top of the Stellar network, we recommend that you initially
```

Comment:
I'd get rid of the 2nd creating:

When creating your application on top of the Stellar network, we recommend starting on the testnet, and migrating to pubnet after rigorous testing has proved it to be production ready.

Reply:
Thanks for all of these suggestions Justin! Since you've been writing a lot of your feedback as as directly applicable suggestions, as a pro tip you can insert suggestions into Github for me to automatically accept. The format is below, or you can see it in the rich text as the `+/-` in the page icon to the left of the font size, bold, and italic options.
```
    ```suggestion
    When creating your application on top of the Stellar network, we recommend that you initially
    ```
```